image/image_linux.go

```diff
@@ -508,16 +508,20 @@ var setupSeed = func(tsto *tooling.ToolingStore, model *asserts.Model, opts *Opt
 	}
 
 	if opts.Classic {
-		// TODO:UC20: consider Core 20 extended models vs classic
-		seedFn := filepath.Join(seedDir, "seed.yaml")
+		var fpath string
+		if core20 {
```

Comment:
in this case we might want the optiion not to exist early if we are building a core-lime image with an ubuntu-seed. We should leave at least a TODO about that

Reply:
Thanks, added TODO and a unit test.